src/app/sidenav/sidenav.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { AudioService } from '../services/audio.service';
-import { faFire, faSearch, faTools } from '@fortawesome/free-solid-svg-icons';
+import { faFire, faBolt faSearch, faTools } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Here's a missing comma but those things happen when you can't build the project so it's understandable!